### PR TITLE
Class Level Stats System for creatures

### PR DESCRIPTION
**Pull request by geo-tp**
Creature classlevelstats
===============

- Implementation of Vmangos classlevelstats to avoid raw values and use multipliers instead
- Specific stats for each unit_class/level
- Multipliers are still at default values atm, i will parse them from Vmangos later

### CODE REVIEW

File: game/world/managers/objects/units/creature/CreatureManager.py [hunk 1]
```diff
@@ -77,35 +77,70 @@ def __init__(self, **kwargs):
...
+            creature_template.damage_multiplier,
+            creature_template.damage_variance
+        )
+
+        # Stats
+        self.dmg_min = melee_dmg_min
+        self.dmg_max = melee_dmg_max
+        self.ranged_dmg_min = ranged_dmg_min
+        self.ranged_dmg_max = ranged_dmg_max
+        self.resistance_0 = int(creature_class_level_stats.armor * creature_template.armor_multiplier)
```
Comment (ratkosrb) [inline game/world/managers/objects/units/creature/CreatureManager.py:127]: agility should be added to the total armor value 1 to 1

Comment (GrenderG) [reply game/world/managers/objects/units/creature/CreatureManager.py:127]: image
Seems like agility didn't increase armor in 0.5.3, at least for players. :thinking:

Comment (ratkosrb) [reply game/world/managers/objects/units/creature/CreatureManager.py:127]: Do we have any Beast Lore screenshots from alpha? We could see if armor includes agility or not from that.

Comment (GrenderG) [reply game/world/managers/objects/units/creature/CreatureManager.py:127]: Don't think so, Beast Lore didn't even exist in 0.5.3.

**Pull request merged**